### PR TITLE
chore: add linter & fix errors

### PR DESCRIPTION
**Pull request by Kabir-Ivan**
Enable linter for .jsx, .ts and .tsx.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     "e2e:generate-fixtures": "npm run --workspace=test/func/fixtures generate",
     "e2e:test": "npm run --workspace=test/func/tests test",
     "e2e": "npm run e2e:build-packages && npm run e2e:generate-fixtures ; npm run e2e:test",
-    "lint": "eslint .",
+    "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
```
Comment (KuznetsovRoman) [inline package.json:29]: Can we specify this in `.eslintrc.js` itself?

File: test/unit/lib/static/components/section/title/browser.jsx [hunk 2]
```diff
@@ -103,7 +103,7 @@ describe('<BrowserTitle/>', () => {
             const browsersStateById = {'yabro': {checkStatus: UNCHECKED}};
             const tree = mkStateTree({browsersById, resultsById, browsersStateById});
 
-            const component = mkBrowserTitleComponent({browserId: 'yabro', browserName: 'yabro'}, {tree});
+            mkBrowserTitleComponent({browserId: 'yabro', browserName: 'yabro'}, {tree});
```
Comment (shadowusr) [inline test/unit/lib/static/components/section/title/browser.jsx:106]: Does this line do anything? As far as I can tell we are just callling a function that returns a value and ignore the value.

File: test/unit/lib/static/components/modals/screenshot-accepter/index.jsx [hunk 3]
```diff
@@ -35,6 +35,9 @@ describe('<ScreenshotAccepter/>', () => {
     };
 
     const mkScreenshotAccepterComponent = (props = {}, initialState = {}) => {
+        if (!global.Element.prototype.scrollTo) {
```
Comment (shadowusr) [inline test/unit/lib/static/components/modals/screenshot-accepter/index.jsx:38]: Hm. How did it work before?

File: test/unit/lib/static/components/controls/run-button.jsx [hunk 4]
```diff
@@ -116,7 +116,7 @@ describe('<RunButton />', () => {
 
         it('should switch to "Run checked tests" if there are checked tests', () => {
             selectorsStub.getCheckedTests.returns([{testName: 'testName', browserName: 'browserName'}]);
-            const component = mkConnectedComponent(<RunButton />, {
+            mkConnectedComponent(<RunButton />, {
```
Comment (shadowusr) [inline test/unit/lib/static/components/controls/run-button.jsx:119]: does this line do anything?

**Review by shadowusr — APPROVED** [review]
Besides a few minor issues, looks good to me

File: test/unit/lib/static/components/section/title/browser.jsx (hunk 2 above)
Comment (Kabir-Ivan) [reply test/unit/lib/static/components/section/title/browser.jsx:106]: Yes. It calls a stub that is used in assert inside the component.

File: test/unit/lib/static/components/controls/run-button.jsx (hunk 4 above)
Comment (Kabir-Ivan) [reply test/unit/lib/static/components/controls/run-button.jsx:119]: Yes. It calls a stub that is used in assert inside the component.

**Pull request merged**